sphinx_autorun/pycon.py

```diff
@@ -12,7 +12,9 @@ def main():
     try:
         while True:
             source = next(source_lines)
-            print('>>>', source)
+            # Allow the user to ignore specific lines of output.
+            if '#ignore' not in source:
```

Comment:
I'd use `if not source.endswith('# ignore')` here, to make sure this is at the end of the lay (and not, say, inside a string, etc).